airbyte-db/lib/src/main/resources/configs_database/schema_dump.txt

```diff
@@ -1,3 +1,42 @@
+create table "public"."actor"(
+  "id" uuid not null,
+  "workspace_id" uuid not null,
+  "actor_definition_id" uuid not null,
+  "name" varchar(256) not null,
+  "configuration" jsonb not null,
+  "actor_type" actor_type not null,
+  "tombstone" bool not null,
+  "created_at" timestamptz(35) not null,
+  "updated_at" timestamptz(35) not null,
+  constraint "actor_pkey"
+    primary key ("id")
+);
+create table "public"."actor_definition"(
+  "id" uuid not null,
+  "name" varchar(256) not null,
+  "docker_repository" varchar(256) not null,
+  "docker_image_tag" varchar(256) not null,
+  "documentation_url" varchar(256) not null,
+  "icon" varchar(256) null,
+  "actor_type" actor_type not null,
+  "source_type" source_type null,
+  "spec" jsonb not null,
+  "created_at" timestamptz(35) not null,
+  "updated_at" timestamptz(35) not null,
+  constraint "actor_definition_pkey"
+    primary key ("id")
+);
+create table "public"."actor_oauth_parameter"(
+  "id" uuid not null,
+  "workspace_id" uuid null,
+  "actor_definition_id" uuid not null,
+  "configuration" jsonb not null,
+  "actor_type" actor_type not null,
+  "created_at" timestamptz(35) not null,
+  "updated_at" timestamptz(35) not null,
+  constraint "actor_oauth_parameter_pkey"
+    primary key ("id")
+);
 create table "public"."airbyte_configs"(
```

Comment:
after this migration we can drop this table, right? totally fine if we don't do it in this migration.

Reply:
Yeah I was thinking cleaning up the table as part of a separate migration. I thought it would be more safe